addons/api/addon/handlers/sqlite-handler.js

```diff
@@ -4,16 +4,193 @@
  */
 
 import { getOwner, setOwner } from '@ember/owner';
+import { service } from '@ember/service';
+import { modelMapping } from 'api/services/sqlite';
+import { get } from '@ember/object';
+import { typeOf } from '@ember/utils';
+import { hashCode } from '../utils/hash-code';
+import { generateSQLExpressions } from '../utils/sqlite-query';
 
 export default class SqliteHandler {
+  @service sqlite;
+
+  batchLimit = 5_000;
```

Comment:
Asking for learning purposes: I do understand batchLimit being a property only used within this class, seems to me is declared as type number, but never seen `5_000` to declare `5000` what is the difference?

Reply:
This was originally used in the `indexed-db-handler` for 10,000 so maybe it's less necessary now but this is just a [numeric separator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators) which helps with readability (also see the original [proposal](https://github.com/tc39/proposal-numeric-separator), this was added in ECMAScript 2021). They're functionally the same thing.